src/AnalysisTools.jl

```diff
@@ -0,0 +1,155 @@
+"""
+    AnalysisTools
+
+Module for analyzing the output of the sampling.
+"""
+module AnalysisTools
+
+using DataFrames
+using CSV
+
+export read_output
+export ωᵢ, partition_function, internal_energy, cv
+
+"""
+    read_output(filename::String)
+
+Reads the output file and returns a DataFrame.
+"""
+function read_output(filename::String)
+    return DataFrame(CSV.File(filename))
+end
+
+"""
+    ωᵢ(iters::Vector{Int}, n_walkers::Int)
+
+Calculates the \$\\omega\$ factors for the given number of iterations and walkers.
+The \$\\omega\$ factors account for the fractions of parameter-space volume sampled during
```

Comment:
"parameter-space volume" has a "pure statistics" ring to it. Since we're focused on chemical and physical systems, shouldn't this just be "phase-space volume?"

Reply:
Done!